docs/sql-ref-syntax.md

```diff
@@ -19,7 +19,4 @@ license: |
   limitations under the License.
 ---
 
-Spark SQL is a Apache Spark's module for working with structured data.
-This guide is a reference for Structured Query Language (SQL) for Apache 
-Spark. This document describes the SQL constructs supported by Spark in detail
-along with usage examples when applicable.
+Spark SQL is Apache Spark's module for working with structured data. The SQL Syntax section describes the SQL syntax in detail along with usage examples when applicable.
```

Comment:
@huaxingao can we add the hyperlink in this main page?

Just need to list the following links in this page

- Data Definition Statements
- Data Manipulation Statements
- Data Retrieval(Queries)
- Auxiliary Statements